proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -52,6 +46,7 @@ message ConnectCluster {
     string commit = 2;
     string kafka_cluster_id = 3;
   }
+  // Unique name of connect cluster. For Redpanda Cloud, the value is `redpanda`.
   string name = 1;
   string address = 2;
```

Comment:
What does `address` refer to?

Reply:
This is the host address of the Kafka Connect cluster.

Reply:
```suggestion
  // The host address of the Kafka Connect cluster.
  string address = 2;
```